sdk/synapse/Microsoft.Azure.Management.Synapse/tests/ScenarioTests/WorkspaceOperationTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System.Collections.Generic;
+using Xunit;
+using System.Threading;
+
+namespace Microsoft.Azure.Management.Synapse.Tests
+{
+    public class WorkspaceOperationTests : SynapseManagementTestBase
+    {
+        [Fact]
+        public void TestWorkspaceLifeCycle()
+        {
+            TestInitialize();
+
+            // create workspace
+            string workspaceName = TestUtilities.GenerateName("synapsesdkworkspace");
+            var createParams = CommonData.PrepareWorkspaceCreateParams();
+            var workspaceCreate = SynapseClient.Workspaces.CreateOrUpdate(CommonData.ResourceGroupName, workspaceName, createParams);
+            Assert.Equal(CommonTestFixture.WorkspaceType, workspaceCreate.Type);
+            Assert.Equal(workspaceName, workspaceCreate.Name);
+            Assert.Equal(CommonData.Location, workspaceCreate.Location);
+            for(int i=0; i<60; i++)
+            {
+                var workspaceGet = SynapseClient.Workspaces.Get(CommonData.ResourceGroupName, workspaceName);
+                if (workspaceGet.ProvisioningState.Equals("Succeeded"))
+                {
+                    Assert.Equal(CommonTestFixture.WorkspaceType, workspaceGet.Type);
+                    Assert.Equal(workspaceName, workspaceGet.Name);
+                    Assert.Equal(CommonData.Location, workspaceGet.Location);
+                    break;
+                }
+
+                Thread.Sleep(30000);
+                Assert.True(i < 60, "Synapse Workspace is not in succeeded state even after 30 min.");
```

Comment:
>min [](start = 95, length = 3)

seconds?

Reply:
The information will be shown when the condition is false. In that case, i = 60, that means `60*30000ms=>60*30s=>30mins`.

Reply:
Alright. Thanks for the explanation..